transforms/ember-object/__testfixtures__/decorators.output.js

```diff
@@ -1,21 +1,21 @@
-import { attribute, className, classNames, layout, tagName } from "@ember-decorators/component";
+import { attribute, className, classNames, tagName, layout as templateLayout } from "@ember-decorators/component";
 
 import {
   filter,
   map as computedMap,
   sum as add,
-  overridableReads as enoWay,
-  overridableReads,
+  oneWay as enoWay,
```

Comment:
enoWay? Looks like a typo from the old output

Reply:
Not a typo, just a local variable name `one` reversed to `eno`